cURL/Complex Flow Examples/decrypt-add-reencrypt.sh

```diff
@@ -40,3 +40,17 @@ curl -X POST "https://api.pdfrest.com/encrypted-pdf" \
   -F "id=$ADDED_IMAGE_ID" \
   -F "output=example_out" \
   -F "new_open_password=password"
+
+
+# All files uploaded or generated are automatically deleted based on the 
+# File Retention Period as shown on https://pdfrest.com/pricing. 
+# For immediate deletion of files, particularly when sensitive data 
+# is involved, an explicit delete call can be made to the API.
+
+# The following code is an optional step to delete decrypted files from pdfRest servers.
+
+curl -X POST "https://api.pdfrest.com/delete" \
+  -H "Accept: application/json" \
+  -H "Content-Type: multipart/form-data" \
+  -H "Api-Key: $API_KEY" \
+  -F "ids=$DECRYPTED_ID, $ADDED_IMAGE_ID"
```

Comment:
This doesn't seem to delete the output from the last curl command where the file was reencrypted